po/fr.po

```diff
@@ -2202,7 +2202,7 @@ msgid ""
 "Space separated list of file patterns used for the find in files dialog (e."
 "g. *.c *.h)"
 msgstr ""
-"Liste de motifs de fichier, séparés par une espace, à utiliser dans le "
+"Liste de motifs de fichier, séparés par un espace, à utiliser dans le "
```

Comment:
Quite ambiguous, `un` refers to the space we see whereas `une` refers to the physical chain which will be inserted. I learned one more thing today.